common/winobjc.nuproj.common.targets

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Target Name="DisableVcxprojPack" BeforeTargets="BeforeResolveReferences" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true' AND '$(Platform)' == 'AnyCPU'">
+    <ItemGroup>
+      <_ProjectReferencesWithPack Include="@(ProjectReference->HasMetadata('Pack'))" />
+      <_ProjectReferencesWithoutPack Include="@(ProjectReference)" />
+      <_ProjectReferencesWithoutPack Remove="@(_ProjectReferencesWithPack)"/>
```

Comment:
Since you are only using `_ProjectReferencesWithPack` here, can you move its `Include` expression into the `Remove` here and get rid of the temporary?

Reply:
I would prefer to do each one explicitly when messing with these project references to ensure ordering.